docs/docs-content/clusters/edge/edge-configuration/installer-reference.md

```diff
@@ -113,21 +113,25 @@ The `stylus.site` blocks accept the following parameters.
-| `prefix` | The system assigns a prefix value to the device UID generated by the Edge Installer. By default, this value is set to `edge`. |
+| `name` | Hostname of the Edge device. A hostname is composed of one label or a series of labels concatenated with dots. For example, `spectro-host` and `host.spectrocloud.com` are both valid host names. The entire hostname, including the delimiting dots, has a maximum of 253 ASCII characters. Each label may contain only the lower-case ASCII letters a through z, the digits 0 through 9, and the hyphen-minus character ('-'), and may contain no more than 63 characters. A hostname must start and end with alphanumeric characters. \
+| `prefix` | A prefix prepended to the Edge device hostname to form the Edge device ID. Only alphanumeric characters and the hyphen-minus character are allowed. By default, this value is set to `edge`. |
 | `network` | The network configuration settings. Review the [Site Network Parameters](#site-network-parameters) below for more details. | 
 | `registrationURL` | The URL that operators should use to register the Edge host with Palette. | 
-| `insecureSkipVerify` | This controls whether or not a client verifies the server's certificate chain and host name. | 
+| `insecureSkipVerify` | This controls whether or not a client verifies the server's certificate chain and hostname. | 
 | `caCerts` | The Secure Sockets Layer (SSL) certificate authority (CA) certificates.| 
-| `clusterId` | The id of the host cluster the edge host belongs to. | 
+| `clusterId` | The ID of the host cluster the edge host belongs to. | 
 | `clusterName` | The name of the host cluster the edge host belongs to. |
 | `tags` | A parameter object you use to provide optional key-value pairs. Refer to the [Tags](#tags) section to learn more. | 
 | `tagsFromFile` | Specify tags from a file. Refer to the [Tags](#tags) section to learn more. | 
 | `tagsFromScript` | Use a script to generate the tags. Refer to the [Tags](#tags) section to learn more. |  
 | `deviceUIDPaths` | Specify the file path for reading in product or board serial that can be used to set the device ID. The default file path is **/sys/class/dmi/id/product_uuid**. Refer to the [Device ID (UID) Parameters](#device-id-uid-parameters) section to learn more.|
 
+:::info
+If you do not specify an hostname for the edge device, the system will generate one from the serial number of the device. If Stylus is unable to identify the serial number, it will generate a random ID instead. In cases where the hardware does not have a serial number, we suggest that you specify a value so there is minimal chance of duplication. Use the value `"$random"` to generate a random ID. You can also use the `DeviceUIDPaths` to read in a value from a system file. 
```

Comment:
```suggestion
If you do not specify a hostname for the edge device, the system will generate one from the serial number of the device. If the Edge Installer cannot identify the serial number, it will generate a random ID instead. In cases where the hardware does not have a serial number, we suggest that you specify a value so there is minimal chance of duplication. Use the value `"$random"` to generate a random ID. You can also use the `DeviceUIDPaths` to read in a value from a system file. 
```
Stylus is an internal name, and we don't want to expose it externally. Although we do a poor job of it, but that's the guidance from the product team and marketing. Use edge installer instead.